src/locales/en/main.json

```
@@ -2575,6 +2579,20 @@
     "viewMoreDetailsPlans": "View more details about plans & pricing",
     "nextBillingCycle": "next billing cycle",
     "yourPlanIncludes": "Your plan includes:",
+    "whatsIncluded": "What's included:",
```

Biome (2.5.0)

[error] The key whatsIncluded was already declared.

(lint/suspicious/noDuplicateObjectKeys)